week-4/F-object-keys/exercise-part-2.js

```diff
@@ -0,0 +1,46 @@
+/*
+
+Objects very often contain other objects. You'll encounter nested objects all the time in the wild. This exercise will give you practice in getting the keys of objects inside other objects.
+
+Use the provided console.log statements below and follow the instructions above them.
```

Comment:
I think the students might need some extra guidance here. There aren't really instructions. You could perhaps say something like "Use the keys() method on the `storeBranches` object and complete the log statements to match the expected output in each case".